# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/encoding_glue/Cargo.toml
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/intl/encoding_glue/Cargo.toml b/intl/encoding_glue/Cargo.toml
--- a/intl/encoding_glue/Cargo.toml
+++ b/intl/encoding_glue/Cargo.toml
@@ -7,9 +7,9 @@ license = "MIT/Apache-2.0"
 
 [features]
 simd-accel = ["encoding_rs/simd-accel"]
 
 [dependencies]
 encoding_rs = "0.8.0"
 nsstring = { path = "../../xpcom/rust/nsstring" } 
 nserror = { path = "../../xpcom/rust/nserror" }
-xmldecl = "0.1.1"
+xmldecl = "0.2.0"